### PR TITLE
Update remove-former-employee-step-4.md

### PR DESCRIPTION
**Pull request by msbemba**
Added an important note about an unsupported scenario

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11272

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/remove-former-employee-step-4.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ Follow these steps to [convert the user's mailbox to a shared mailbox](../email/
 5. Select **Save**.
 6. Remember, don't delete the former employee's account.
 
+> [!IMPORTANT]
+> In Scenarios where in the mailbox is Inactive for complaince reasons, the mail forwarding or converting it to a shared mailbox would not work.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee-step-4.md]:
```suggestion
> In scenarios where the mailbox is inactive due to compliance reasons, mail forwarding or converting the mailbox to a shared mailbox would not work.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**